rotkehlchen/db/history_events.py

```diff
@@ -66,15 +66,19 @@
 
 def maybe_filter_ignore_asset(
         filter_query: HistoryBaseEntryFilterQuery,
+        negation: bool = False,
```

Comment:
negation is a really bad argument name here. The reason is that is very generic and is not documented in the docstring. If you are the person that reads this code negation doesn't mean anything to you and is confusing making you go to the function and read how it works.

What do you thing of using `include_ignored_assets`? Also document it in the docstring